common/log.cpp

```diff
@@ -34,7 +34,8 @@ static void LogStdOut(
     const char* message
     )
 {
-    printf("[LOG %u] %s", level, message);
+    fprintf(stderr, "[LOG %u] %s", level, message);
+    fflush(stderr);
```

Comment:
in theory... in practice i've had problems. there were some std::abort calls that were not getting the output from the log even when pushing stderr. it may be superflous though i would prefer to keep it.

Reply:
definitely safe, was just curious ....